controllers/purchaseorder.js

```diff
@@ -0,0 +1,224 @@
+var _ = require('lodash'),
+	crmPrivileges = require('../utils/crm_privileges'),
+	modulesConfig = require('../config/modules_config'),
+	async = require('async'),
+	pagination = require('../utils/pagination'),
+	common = require('../utils/common');
+/**
+ * @swagger
+ * resourcePath: /quotes
+ * description: sQcrm Quotes 
+ */
+module.exports = function(app, config) {
+	var apiNamespace = config.apiNamespace,
+		resourceType = 'purchaseorders';
+	
+	// Public Methods
+	return {
+		/**
+		* @swagger
+		* path: /api/v1/purchaseorders
+		* operations:
+		*   -  httpMethod: GET
+		*      summary: Get all the purchaseorders
+		*      notes: Return all the purchaseorders associated with the token
+		*      responseClass: purchaseorder
+		*      nickname: purchaseorder
+		*      parameters:
+		*        - name: Authorization
+		*          description: access token
+		*          paramType: header
+		*          required: true
+		*          dataType: string
+		*          defaultValue: Bearer ACCESS_TOKEN_HERE
+		*        - name: page
+		*          description: page number of results
+		*          paramType: query
+		*          required: false
+		*          dataType: integer
+		*          defaultValue: 1
+		*        - name: limit
+		*          description: the number of results per page
+		*          paramType: query
+		*          required: false
+		*          dataType: integer
+		*          defaultValue: 50
+		*/
+		getAll: function(req, res, next) {
+			var apiBase = req.protocol + '://' + req.get('host') + apiNamespace,
+				apiEndpoint = apiBase + '/' + resourceType + '/',
+				whereClause,
+				page,
+				limit;
+
+			crmPrivileges.userWhereCondition(req, 16, 'PurchaseOrder', 'purchase_order_custom_fld', true, function(err, whereCond) {
```

Comment:
The 2nd param in the above function is the table name https://github.com/sQcrm/sqcrm-node-api/blob/pruchase_order/utils/crm_privileges.js#L285 
You are sending the module name, the table name is `purchase_order`